File: Basic-Programs/Program19.java
import java.util.Scanner;

class Demo
{
    public float fPercentage = 0.0f;
    
    Demo(float fNo)
    {
        fPercentage = fNo;
    }
    
    void DisplayResult()
    {
        if((fPercentage >= 0.0f)  &&  (fPercentage <= 100.0f))
        {
            if(fPercentage >= 35)
            {
                System.out.println(" You Are Pass ");
            }
            else
            {
                System.out.println(" You Are Fail ");
            }   
        }
        else
        {
            System.out.println("Invalid Percentage. please Enter the Value in Range 0 To 100");
            return;
        }
    }
}

class Program19
{
    public static void main(String A[])
    {
        float fValue = 0.0f;
        
        Scanner Sc = new Scanner(System.in);
        System.out.println(" Enter Your Percentage ");
        fValue = Sc.nextFloat();
        
        Demo dobj = new Demo(fValue);
        dobj.DisplayResult();
        
    }
}

File: Basic-Programs/Program19.c
#include<stdio.h>

void DisplayResult(float fPercentage)
{
    if(fPercentage >= 0.0f && fPercentage <= 100.0f)        // Filter 
    {
        if(fPercentage >= 40)
        {
            printf("You Are Pass \n");
        }
        else
        {
            printf("You Are Fail \n");
        }
    }
    else
    {
        printf("Invalid Percentage. Please Enter the Value in Range 1 To 100 \n");
    }
}

int main()
{
    float fResult = 0.0f;
    
    printf("Enter The Percentage \n");
    scanf("%f",&fResult);
    
    DisplayResult(fResult);

    return 0;
}